test/test.js

```diff
@@ -491,6 +491,19 @@ suite('Request', function() {
     })
   })
 
+  test('clone request bodies', function() {
+    var r1 = new Request('https://fetch.spec.whatwg.org/', {
+      method: 'post',
+      headers: {'content-type': 'text/plain'},
+      body: 'I work out'
```

Comment:
Well, it wouldn't be testing the browser implementation so much as testing that you can clone a Request with a Blob body and read that body uninterrupted from both clones.